web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -456,8 +458,20 @@ export const BusinessFormationPaginator = (): ReactElement => {
           <div>{Config.formation.errorBanner.errorOnStep}</div>
           <ul>
             {dedupedFieldErrors.map((fieldError) => {
+              let configFieldName = fieldError.field as ConfigFormationFields;
+
+              if (fieldError.field === "members") {
+                const isCorp = ["c-corporation", "s-corporation"].includes(state.formationFormData.legalType);
```

Comment:
We should use the existing array from formationData `corpLegalStructures` instead of redeclaring this here

Reply:
Done